chain/indexer.go

```diff
@@ -183,192 +183,189 @@ func (t *TipSetIndexer) TipSet(ctx context.Context, ts *types.TipSet) error {
 	// A map to gather the persistable outputs from each task
 	taskOutputs := make(map[string]model.PersistableList, len(t.processors)+len(t.actorProcessors))
 
-	// Run each tipset processing task concurrently
-	for name, p := range t.processors {
-		inFlight++
-		go t.runProcessor(tctx, p, name, ts, results)
-	}
-
-	// Run each actor or message processing task concurrently if we have any and we've seen a previous tipset to compare with
-	if len(t.actorProcessors) > 0 || len(t.messageProcessors) > 0 || len(t.messageExecutionProcessors) > 0 || len(t.consensusProcessor) > 0 {
-
-		// Actor processors perform a diff between two tipsets so we need to keep track of parent and child
-		var parent, child *types.TipSet
-		if t.lastTipSet != nil {
-			if t.lastTipSet.Height() > ts.Height() {
-				// last tipset seen was the child
-				child = t.lastTipSet
-				parent = ts
-			} else if t.lastTipSet.Height() < ts.Height() {
-				// last tipset seen was the parent
-				child = ts
-				parent = t.lastTipSet
-			} else {
-				log.Errorw("out of order tipsets", "height", ts.Height(), "last_height", t.lastTipSet.Height())
-			}
+	// The parent is the primary tipset that tasks act upon. The child adds additional context such as outcomes of
+	// message execution.
+	var parent, child *types.TipSet
```

Comment:
What do you think of renaming these?
parent -> current
child -> next

Reply:
Love it. I'll make that change.

Reply:
Will also adjust the logging to include this